docs/quickstarts/excel-quickstart-jquery-experiment.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Build your first Excel task pane add-in
+description: Learn how to build a simple Excel task pane add-in by using the Office JS API.
+ms.date: 12/11/2023
+ms.service: excel
+ms.localizationpriority: high
+---
+
+# Build an Excel task pane add-in
+
+In this article, you'll walk through the process of building an Excel task pane add-in.
+
+## Create the add-in
+
+[!include[Choose your editor](../includes/quickstart-choose-editor-experiment.md)]
+
+# [One-line script](#tab/script)
+
+### Prerequisites
+
+You must have Office connected to a Microsoft 365 subscription.
+
+> [!NOTE]
+> If you don't already have Office, you can [join the Microsoft 365 developer program](https://developer.microsoft.com/microsoft-365/dev-program) to get a free, 90-day renewable Microsoft 365 subscription to use during development.
+
+### Create the add-in project
+
+You can run the one-line script on Windows or on macOS. It performs the following to set up the add-in development environment.
+- Installs Node.js, npm, and Git, if they aren't found on your machine.
+- Downloads the files of the sample add-in project.
+- Opens the add-in project in Visual Studio Code, if installed. If Visual Studio Code isn't installed on your machine, the folder that contains the add-in project files opens instead.
+- Prompts you to run the add-in.
+> [!NOTE]
+> In addition to Windows and macOS, we're working to support the script on other platforms.
+
+If you're using Windows, run the following from a command prompt.
+```command&nbsp;line
+iwr aka.ms/exceladdin/helloworld -o excelhelloworld.bat; saps excelhelloworld.bat
+```
+If you're using macOS, run the following in a terminal.
+```command&nbsp;line
+bash <(curl –s –L aka.ms/exceladdin/helloworld_mac) 
+```
+
+### Try it out
+
+Once your add-in project is set up, it automatically opens in Visual Studio Code. If you don't have Visual Studio Code installed on your machine, the folder containing the files of your add-in project opens instead.
```

Comment:
```suggestion
Once your add-in project is set up, it automatically opens in Visual Studio Code.

> [!NOTE]
> If you don't have Visual Studio Code installed on your machine, the folder containing the files of your add-in project opens instead. For guidance on how to install Visual Studio Code and test the add-in, see the project's readme file. 
```
@GavinGu07 suggesting this to provide additional guidance for users without VS Code installed. I took this from the spec, but let me know if this is no longer accurate.